docs/script.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * Initialises listeners, gathers inputs from the user,
```

Comment:
don't say "initi... listener" cos obvs, but say "this is the main client-side script"

Reply:
Resolved in 8db4fd23338964f92f57384d9a066475495a9679